modular_skyrat/modules/colony_fabricator/code/machines/power_storage_unit.dm

```diff
@@ -0,0 +1,76 @@
+/obj/machinery/power/smes/battery_pack
+	name = "stationary battery"
+	desc = "An about table-height block of power storage, commonly seen in low storage high output power applications. \
+		Smaller units such as these tend to have a respectively <b>smaller energy storage</b>, though also are capable of \
+		<b>higher maximum output</b> than some larger units. Most commonly seen being used not for their ability to store \
+		power, but rather for use in regulating power input and output."
+	icon = 'modular_skyrat/modules/colony_fabricator/icons/power_storage_unit/small_battery.dmi'
+	capacity = 75e4
+	input_level_max = 4e5
+	output_level_max = 4e5
```

Comment:
I didn't even know we had notation like this in DM, and I'm not sure that I'm a fan of it either. I think I'll check in with the other maintainers to see if we want to allow such a notation in the code or not, since it's the first time it's come up.

Reply:
I just used it because that's what every other power machine in the code with large numbers like this uses.

Reply:
In that case, I guess we'll leave it. I still don't really like it, but I'd rather keep things consistent.